news/2019-10-01-new-featured-artist-updates.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title: "Featured Artist Track Updates: October 2019"
+date: 2019-10-01 10:00:00 +0000
+---
+
+We promised a big update in the last Featured Artist reveal, and boy did we mean it. **197** new tracks from **12** of our absurdly talented Featured Artists are now available. Check it all out!
+
+There's so much to go over, so we'll dip right into it - just remember that **clicking the artist banners will take you directly to their Featured Artist listing**, and that the newer tracks listed in this post are likely to be towards the bottom of their respective page.
```

Comment:
```suggestion
There's so much to go over, so we'll dip right into it—just remember that **clicking the artist banners will take you directly to their Featured Artist listing**, and that the newer tracks listed in this post are likely to be towards the bottom of their respective page.
```

Reply:
i'll do this one since it's technically correct but man i swear the spaced hyphen looks way better than a joining em dash

Reply:
familiar things look better
so do it the right way and eventually this will look good too! :^)